internal/engine/planner/planner.go

```diff
@@ -538,6 +544,10 @@ func (p *partialEvaluator) evalPartially(e *exprpb.Expr) (ref.Val, *exprpb.Expr,
 func newEvaluator(input *enginev1.PlanResourcesInput) (p *partialEvaluator, err error) {
 	p = new(partialEvaluator)
 	knownVars := make(map[string]any)
+	knownVars[conditions.CELRequestIdent] = input
```

Comment:
I am probably not understanding this correctly. Is it correct to say `request` is equal to the `PlanResourcesInput`? Normally `request` is `engine.CheckInput`, isn't it?

Reply:
Yes and yes. This code path belongs exclusively to the query planner API. In this case `input` is `enginev1.PlanResourcesInput` as per https://github.com/cerbos/cerbos/blob/main/internal/svc/cerbos_svc.go#L60.